travis/test_server.py

```diff
@@ -296,11 +304,36 @@ def main(argv=None):
     counted_errors = 0
     for to_test in to_test_list:
         print("\nTesting %s:" % to_test)
-        subprocess.call(["createdb", "-T", dbtemplate, database])
+        db_odoo_created = False
+        try:
+            db_odoo_created = subprocess.call(
+                ["createdb", "-T", dbtemplate, database])
+        except subprocess.CalledProcessError:
+            db_odoo_created = True
         for command, check_loaded in commands:
-            command[-1] = to_test
-            # Run test command; unbuffer keeps output colors
-            command_call = ["unbuffer"] + command
+            if db_odoo_created and instance_alive:
+                # If exists database of odoo test
+                # then start server with regular command without tests params
+                command_start = commands[0][0]
+                rm_items = [
+                    'coverage', 'run', '--stop-after-init',
+                    '--test-enable', '--init', None,
+                    '--log-handler', 'openerp.tools.yaml_import:DEBUG',
+                ]
+                for rm_item in rm_items:
+                    try:
+                        command_start.remove(rm_item)
+                    except ValueError:
+                        pass
```

Comment:
Replace the 6 line for with a 1 line list comprehension: 
`command_start = [x for x in command[0] if x not in rm_items]`

Nevertheless, this feels like a hot fix and not evolution friendly.
Isn't there a way to we handle this above, when we construct the command?